### PR TITLE
[CONFLICT?] Pixel art graphics for inventory sprites

### PR DESCRIPTION
**Pull request by ComputerOverlord**
Pretty much summed up in the title, Also this is my first PR.

### CODE REVIEW

**Comment by ComputerOverlord** [comment]
I'm going to go into unity and post images

File: Assets/StreamingAssets/Images/Inventory/Steel Plate.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Sprites>
-	<Sprite name="Steel Plate"     x="0" y="0" w="128" h="128" pixelPerUnit="128" />
+<?xml version="1.0" encoding="utf-8"?>
+<Sprites>
+	<Sprite name="Raw Iron"     x="0" y="0" w="7" h="7" pixelPerUnit="7" />
```
Comment (zwrawr) [inline Assets/StreamingAssets/Images/Inventory/Steel Plate.xml]: This is Steel_Plate not Raw_Iron

File: Assets/StreamingAssets/Images/Inventory/Raw Copper.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Sprites>
+	<Sprite name="Raw Iron"     x="0" y="0" w="7" h="7" pixelPerUnit="7" />
```
Comment (zwrawr) [inline Assets/StreamingAssets/Images/Inventory/Raw Copper.xml]: Why is the sprite in Raw_Copper.xml called Raw Iron? Check when copying and pasting

File: Assets/StreamingAssets/Images/Inventory/Copper Wire.xml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Sprites>
-	<Sprite name="Copper Wire"     x="0" y="0" w="128" h="128" pixelPerUnit="128" />
+<?xml version="1.0" encoding="utf-8"?>
+<Sprites>
+	<Sprite name="Raw Iron"     x="0" y="0" w="7" h="7" pixelPerUnit="7" />
```
Comment (zwrawr) [inline Assets/StreamingAssets/Images/Inventory/Copper Wire.xml]: This is Copper Wire not Raw Iron

**Comment by ComputerOverlord** [comment]
Umm...

**Pull request closed without merging**